cms/models.py

```diff
@@ -12,6 +13,65 @@
 from wagtail.images.blocks import ImageChooserBlock
 
 
+class CourseProgramChildPage(Page):
+    """
+    Abstract page representing a child of Course/Program Page
+    """
+
+    class Meta:
+        abstract = True
+
+    parent_page_types = ["CoursePage", "ProgramPage"]
+
+    @classmethod
+    def can_create_at(cls, parent):
+        # You can only create one of these page under course / program.
+        return (
+            super(CourseProgramChildPage, cls).can_create_at(parent)
+            and parent.get_children().type(cls).count() == 0
+        )
+
+
+class WhoShouldEnrollPage(CourseProgramChildPage):
+    """
+    Who should enroll child page for "Who Should Enroll" section.
+    """
+
+    subpage_types = []
+
+    image = models.ForeignKey(
+        Image,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        help_text="Image size must be at least 870x500 pixels.",
+    )
+    content = StreamField(
+        [
+            (
+                "item",
+                blocks.RichTextBlock(
+                    icon="plus", features=["bold", "italic", "ol", "ul"]
+                ),
+            )
+        ],
+        blank=False,
+        help_text='Contents of the "Who Should Enroll" setion.',
+    )
+
+    content_panels = [StreamFieldPanel("content"), FieldPanel("image")]
+
+    # disable promote panels, no need for slug entry, it will be autogenerated
+    promote_panels = []
```

Comment:
should we not move it to parent ? I am thinking we will keep it disable for others too.

Reply:
Done.